app/assets/javascripts/discourse/app/components/user-profile-avatar.hbs

```diff
@@ -1,11 +1,26 @@
-<div class="user-profile-avatar">
+<PluginOutlet
+  @name="user-profile-avatar-wrapper"
+  @connectorTagName="div"
+  @outletArgs={{hash user=@user}}
+>
   {{bound-avatar @user "huge"}}
```

Comment:
you've got two avatars now here 👀 
```suggestion
```

Reply:
This one is wrapped by the plugin outlet so it should be replaced by the plugin

Reply:
I mean there's a second one below